integration_tests/test_str_01.py

```diff
@@ -56,5 +80,5 @@ def check():
     test_str_slice()
     test_str_repeat()
     test_constant_str_subscript()
-
+    test_str_title()
```

Comment:
Let's test other two functions as well, `test_str_istitle` and `test_str_isalpha`